torchvision/transforms/functional_tensor.py

```diff
@@ -0,0 +1,32 @@
+import torch
+import torchvision.transforms.functional as F
+
+def vflip(img_tensor):
+    """Vertically flip the given the Image Tensor.
+
+    Args:
+        img_tensor (Tensor): Image Tensor to be flipped.
+
+    Returns:
+        Tensor:  Vertically flipped image Tensor.
+    """
+    if not F._is_tensor_image(img_tensor):
+    	raise TypeError('tensor is not a torch image.')
+    
+    return img_tensor.flip(1)
```

Comment:
Can you make the offset relative to the end of the dimension?
This way, we could make it also work for batches of images / videos.
So it would be something like

```python
return img_tensor.flip(-2)
```